### ISSUE TITLE
Dockerfile request

### ISSUE THREAD

**Issue by rhelsing** [falkTX/kuriborosu#4]
Hey there, I'd love to help with this project. I'd like to build a dockerfile that can consistently compile the project and optionally include a VST or two to for demonstration. What are your thoughts on this approach?

**Comment by rhelsing** [comment]
Along with this, some clear usage documentation would be helpful, and make this project more approachable.

**Comment by rhelsing** [comment]
Can the library currently do what is claimed on the readme?

**Comment by falkTX** [comment]
> Can the library currently do what is claimed on the readme?

Yes, except it cant specify parameter values from within the plugins loaded.

For the docker part you probably want to build a custom carla instead of the one in the ubuntu repos, so it is more recent.
So building carla can be a first step.
Should be ok following https://github.com/falkTX/Carla/blob/main/.github/workflows/build.yml
